helmrepositories/mesosphere-spark-repo/mesosphere-spark-repo.yaml

```
@@ -0,0 +1,8 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: mesosphere-spark-repo
```

just a note but should be ok to leave `namespace` blank (instead of `namespace: ${releaseNamespace}`) because we define `targetNamespace` on the project kustomization deployed in kommander

---

```suggestion
  name: mesosphere.github.io-spark-charts
```

maybe just to be consistent with our helmrepo namings in kommander